Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml

```
@@ -24,6 +24,8 @@
     ClothingOuterArmorDuraVest: 2
     ClothingHeadHelmetBasic: 2 # DeltaV - added helmets to the SecTech. Another line of defense between the tide and your grey matter.
     # BreachingCharge: 8
+    LPPClothingHandsGlovesTacticalFingerless: 1 # Lost Paradise - Добавление тактических беспалых боевых перчаток
```

yamllint

[warning] too few spaces before comment

(comments)